src/main/java/io/openliberty/tools/intellij/util/LibertyProjectUtil.java

```diff
@@ -157,21 +157,21 @@ public static ShellTerminalWidget getTerminalWidget(Project project, LibertyModu
                                                         TerminalToolWindowManager terminalToolWindowManager, ShellTerminalWidget widget) {
         // Set Terminal engine to CLASSIC
         if (widget == null && createWidget) {
-            try {
-                Class<?> optionsProviderClass = Class.forName("org.jetbrains.plugins.terminal.TerminalOptionsProvider");
-                Object optionsProviderInstance = optionsProviderClass
-                        .getMethod("getInstance")
-                        .invoke(null);
-
-                Class<?> terminalEngineClass = Class.forName("org.jetbrains.plugins.terminal.TerminalEngine");
-                Object classicEngine = Enum.valueOf((Class<Enum>) terminalEngineClass, "CLASSIC");
-                Method setEngineMethod = optionsProviderClass
-                        .getMethod("setTerminalEngine", terminalEngineClass);
-                setEngineMethod.invoke(optionsProviderInstance, classicEngine);
+            if (shouldForceClassicTerminal()) {
+                try {
+                    Class<?> optionsProviderClass = Class.forName("org.jetbrains.plugins.terminal.TerminalOptionsProvider");
+                    Object optionsProviderInstance = optionsProviderClass
+                            .getMethod("getInstance")
+                            .invoke(null);
 
-            } catch (ClassNotFoundException | NoSuchMethodException |
-                     IllegalAccessException | InvocationTargetException e) {
-                LOGGER.error("Failed to set TerminalEngine to CLASSIC via reflection: " + e.getMessage());
+                    Class<?> terminalEngineClass = Class.forName("org.jetbrains.plugins.terminal.TerminalEngine");
+                    Object classicEngine = Enum.valueOf((Class<Enum>) terminalEngineClass, "CLASSIC");
+                    Method setEngineMethod = optionsProviderClass
+                            .getMethod("setTerminalEngine", terminalEngineClass);
+                    setEngineMethod.invoke(optionsProviderInstance, classicEngine);
+                } catch (Exception e) {
+                    LOGGER.info("Falling back to default terminal engine.", e);
```

Comment:
@dessina-devasia I think this should be debug or trace level in the log. I can only imagine this message being useful to debugging control flow through the plugin which is not something a regular user should be concerned about.